app/src/routes/project/LayersPanel/index.tsx

```diff
@@ -45,7 +45,7 @@ const LayersPanel = observer(() => {
                         <PinLeftIcon />
                     </button>
                 </TabsList>
-                <Separator className="mt-1" />
+                <Separator className="mt-1" id="LayersPanelDiv" />
```

Comment:
I would move the id to the tab container since we can use it to calculate the width offset
https://github.com/onlook-dev/onlook/pull/298/commits/09ce88b6e7cc9ecf0f8c24b1dfaa4ad5190777f2#diff-f0245d5cb38ab89312972b0120a231f7e191c7938935b2c5dd18f4c3cddbfe08R49